pkgs/data/themes/pop-gtk/default.nix

```diff
@@ -0,0 +1,80 @@
+{ stdenv
+, fetchurl
+, intltool
+, bash
+, which
+, parallel
+, pkgconfig
+, sassc
+, gtk3
+, inkscape
+, optipng
+, gnome3
+, gtk-engine-murrine
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pop-gtk-theme";
+  version = "3.1.2";
+
+  src = fetchurl {
+    url    = "https://github.com/pop-os/gtk-theme/archive/${version}.tar.gz";
```

Comment:
GitHub archives are sometimes regenerated over time and the hash changes over time, so you shouldn't use `fetchurl` here. Instead, use `fetchFromGitHub` or `fetchzip`.
The hash for those two fetchers needs to be computed over the content of the archive _after_ unpacking. You can precompute it with `nix-prefetch-url --type sha256 unpack <archive_url>`

Reply:
I didn't knew file could changes, I switched to `fetchFromGitHub`, thanks !